Prometheus configuration (#1878)
strimzi/strimzi-kafka-operator: documentation/book/con-metrics-prometheus-resources.adoc

* A `Prometheus` to manage the configuration of the Prometheus pod.
* A `PrometheusRule` to manage alerting rules for the Prometheus pod.
* A `Secret` to manage additional Prometheus settings.

> * A `Service` to provide an easy to reference hostname for other services to connect to Prometheus.

Reviewer: "to allow applications running in the cluster to connect to Prometheus (i.e. Grafana using Prometheus as datasource)"

> * A `Service` to allow applications running in the cluster to connect to Prometheus (i.e. Grafana using Prometheus as datasource).

Reviewer 2: ```suggestion
* A `Service` to allow applications running in the cluster to connect to Prometheus (where Grafana is using Prometheus as data source).
```

Author: @ppatierno are you ok with this?

Reviewer: @PaulRMellor I meant that Grafana is one of the applications that can use Prometheus, reading your suggestion it's not the same.

Author: `docu_check` and therefore entire build because string "i.e." is prohibited.

Reviewer: @PaulRMellor what for saying "for example" ?
